### DISCUSSION TITLE
Bootstrap trees from pseudo-replicates

### DISCUSSION THREAD

**Discussion by dnakamuraz** [iqtree/iqtree2#385] [Q&A]
Hello everyone! Happy New Year!

My question: Is it possible to give as output a tree file containing the multiple bootstrap trees from the pseureplicates? By multiple bootstrap trees, I do not refer to the majority consensus tree wih frequency of clades (which is the output usually desired). I mean the original trees used to calculate the bootstrap frequencies. For instance, if the user requires -b 1000, I guess that at least 1000 trees from 1000 pseudo-replicated matrices were generated in temporary files to calculate the final frequencies.

I would like a tree file containing these pseudo-replicated trees because I want to calculate frequencies using other methods available in R packages.

Thank you very much!

**Comment by bqminh** [comment]
With -b 1000 IQ-TREE will print .boottrees file containing all 1000 bootstrap trees.
If you want to speed up standard bootstrap, here is a FAQ: http://www.iqtree.org/doc/Frequently-Asked-Questions#how-do-i-save-time-for-standard-bootstrap
